executable/microsoft_pe.ksy

```diff
@@ -408,3 +413,134 @@ types:
         size: length - 8
         doc: Contains a certificate, such as an Authenticode signature.
     doc-ref: 'https://docs.microsoft.com/en-us/windows/desktop/debug/pe-format#the-attribute-certificate-table-image-only'
+
+  resource_directory_table:
+    params:
+      - id: depth
+        type: u4
+      - id: section_file_offset
+        type: u4
+      - id: section_virtual_address
+        type: u4
+    seq:
+      # This field is reserved for future use. It is currently set to zero.
+      - id: characteristics
+        type: u4
+      - id: time_date_stamp
```

Comment:
Maybe just established name `timestamp`? `time_date_stamp` seems bloated and weird to me.

It's not necessary at all to strictly follow the original naming from the original spec.

```suggestion
      - id: timestamp
```

Also, what timestamp format does it use?